src/Components/BookDetails.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getSingleBook } from '../Redux/Books/reducerBooks';
+
+function BookDetails() {
+  const dispatch = useDispatch();
+  const param = useParams();
+  const books = useSelector((state) => state.singleBook);
+  const { id } = param;
+
+  useEffect(() => {
+    if (books.length === 0) dispatch(getSingleBook(id));
+  }, []);
+
+  const {
+    name, image, age, gender, hairColor, relatives, occupation, firstEpisode, voiceBy, wikiUrl,
+  } = books;
+
+  return (
+    <div className="single-book__details">
+      <article className="card-Single">
+        <div className="card-img__single">
+          <img src={image} alt="" />
+        </div>
+        <div className="card-body__single">
+          <h2>{name}</h2>
+          <p>
+            Age:
+            {' '}
+            {age}
+          </p>
+          <p>
+            Gender:
+            {' '}
+            {gender}
+          </p>
+          <p>
+            Hair-Color:
+            {' '}
+            {hairColor}
+          </p>
+          <p>
+            Relatives:
+            {' '}
+            {relatives}
+          </p>
+          <p>
+            Occupation:
+            {' '}
+            {occupation}
+          </p>
+          <p>
+            First-Episode:
+            {' '}
+            {firstEpisode}
+          </p>
+          <p>
+            Voice By:
```

Comment:
Kindly make that your layout is consistent, on the local version the details are not displayed properly. The text overflows and the footer overlays with the details.

![image](https://user-images.githubusercontent.com/38088818/195978460-fffc9fb2-afe9-4fd6-8943-23f819012c93.png)

Your live version is a little bit better but the text still overflows.

![image](https://user-images.githubusercontent.com/38088818/195978509-e027d770-5d34-4ffc-b536-48d98feb4908.png)